.github/workflows/ft-sync.yml

```diff
@@ -0,0 +1,26 @@
+on:
```

Comment:
Remove this entire file and put it in your sync demo repo.

Reply:
Yeah doing that only